docs/keystatic.config.tsx

```diff
@@ -45,6 +46,36 @@ export const componentBlocks = {
     },
     chromeless: true,
   }),
+  'cloud-image': component({
+    preview: CloudImagePreview,
+    label: 'Cloud image',
+    schema: {
+      href: fields.text({
+        label: 'Href *',
+        validation: {
+          length: {
+            min: 1,
+          },
+        },
+      }),
+      alt: fields.text({
+        label: 'Alt text',
+      }),
+      height: fields.text({
+        label: 'Height',
+      }),
+      width: fields.text({
```

Comment:
Could potentially be a `fields.integer` cause when playing I tried putting in "12rem" which didn't work (and then I read the code 😅)

Or otherwise maybe we could set the width/height with css instead 🎨

Reply:
Yeah good thinking, I like it 🙌 

The height and width are more to avoid layout shift by hinting to the browser what to expect and reserve that space. I was even thinking we could put these values in an `Advanced features` Accordion with some descriptions or something but I didn't want to get too far ahead of myself because this all really needs some dev/design sync + thoughts.

I will defs do the `fields.integer`, though 🙂